test/onigumo_downloader_test.exs

```diff
@@ -26,17 +26,17 @@ defmodule OnigumoDownloaderTest do
     end
   end
 
-  describe("Onigumo.Downloader.download_urls_from_file/1") do
+  describe("Onigumo.Downloader.create_download_stream/1") do
     @tag :tmp_dir
-    test("download URLs from the input file", %{tmp_dir: tmp_dir}) do
+    test("create a stream to download URLs from the input file", %{tmp_dir: tmp_dir}) do
```

Comment:
```suggestion
    test("download URLs from the input file with a created stream", %{tmp_dir: tmp_dir}) do
```

Reply:
Thank you for the suggestion! I find it good, because this test does not focus on creating the stream, but on using the stream to download the URLs. Although without the function returning a stream, this test would fail on `|> Stream.run()`.

A proper unit test would verify that the stream is composed properly, comparing the actual stream to an expected one. There is a follow-up issue for that: #173.